README.md

```diff
@@ -78,7 +78,7 @@ Rails/FindBy:
 
 ## Documentation
 
-You can read a lot more about RuboCop Rails in its [official docs](https://docs.rubocop.org/projects/rails/).
+You can read a lot more about RuboCop Rails in its [official docs](https://rails.rubystyle.guide/).
```

Comment:
I will soon have documentation available at the following URL (It's still 404):
https://docs.rubocop.org/rubocop-rails/

Reply:
FYI, https://docs.rubocop.org/rubocop-rails/ is now available.